src/components/Theme/style.scss

```diff
@@ -145,6 +145,10 @@ $logo-height: 32 * $unit;
   height: $footer-height;
   flex: 0 0 auto;
 
+  .fullScreenStep & {
+    z-index: 1;
+  }
+
```

Comment:
Does this behave as expected when the SDK is used as a modal?

Reply:
Yes, it does. I'll add a note in the ticket's acceptance criteria about checking for this too & also to check with different browsers like Safari, IE11 which I think was the original reason it was removed. But not sure if it was just because of IE11 or Safari.